src/App.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import './index.css';
+import './App.css';
+import words from './words';
+import images from './images';
+import SoundEffect from './SoundEffect';
+import zvuk1 from './zvuk1.mp3';
+import zvuk2 from './zvuk2.mp3';
+
+const App = () => {
+    const [mainWord, setMainWord] = useState(null);
+    const [randomWords, setRandomWords] = useState([]);
+    const [showMainWord, setShowMainWord] = useState(false);
+    const [showAdditionalWords, setShowAdditionalWords] = useState(false);
+    const [userAnswer, setUserAnswer] = useState('');
+    const [feedback, setFeedback] = useState('');
+    const [mainImage, setMainImage] = useState(null);
+    const winnerSound = new Audio(zvuk1);
+    const lostSound = new Audio(zvuk2)
+
+    const newChangeWord = () => {
```

Comment:
Inappropriate function name. It is not clear what the function is for. 
It should be something like 'showNewQuestionWord', with a verb as the first word in the name

Reply:
`` const showNewQuestionWord = () => {
        const index = Math.floor(Math.random() * words.length);
I fixed everything here

Reply:
I changed the function name to showNewQuestionWord now it is used as it should.
` const [showAdditionalWords, setShowAdditionalWords] = useState(false);`